source/_integrations/blink.markdown

```diff
@@ -90,7 +90,7 @@ Trigger a camera to take a new still image.
 
 | Service Data Attribute | Optional | Description                            |
 | ---------------------- | -------- | -------------------------------------- |
-| `name`                 | no       | Name of camera to take new image with. |
+| `entity_id`            | yes      | Camera entity to take picture with.    |
```

Comment:
`entity_id` is not optional. You need to specify `all` if you want to target all.

Reply:
```suggestion
| `entity_id`            | no       | Camera entity to take picture with.    |
```

Reply:
Oops, Frenck is right! It is optional, but you need to specify either `area_id ` or `entity_id`.